FilterQL.md

```diff
@@ -23,6 +24,7 @@ Comparison    = Identifier ComparisonOp Literal
 ComparisonOp  = ">" | ">=" | "<" | "<=" | "=="
 EXISTS        = "EXISTS" Identifier
 IN            = Identifier "IN" (Literal1, Literal2, ...)
+INTERSECTS    = Identifier "INTERSECTS" (Literal1, Literal2, ...)
```

Comment:
are we sure this is `"INTERSECTS" (Literal1, Literal2, ..) ` vs `"INTERSECTS" (EXPRESSION, EXPRESSION2, ....)`